lib/ocpp/v201/charge_point.cpp

```diff
@@ -1855,16 +1855,22 @@ bool ChargePoint::validate_set_variable(const SetVariableData& set_variable_data
         const auto network_configuration_priorities = ocpp::split_string(set_variable_data.attributeValue.get(), ',');
         const auto active_security_profile =
             this->device_model->get_value<int>(ControllerComponentVariables::SecurityProfile);
+        const auto network_connection_profiles = json::parse(
```

Comment:
This could potentially throw, probably best to catch it here, log a warning/error and return false

Reply:
Addressed this by widening the scope of the try-catch